portal-api/src/v1/facility-dates/requested-cover-start-date.js

```diff
@@ -45,6 +45,7 @@ exports.updateRequestedCoverStartDate = (facility) => {
         requestedCoverStartDateYear,
       )
     );
+    modifiedFacility.coverDateConfirmed = true;
```

Comment:
This is interesting to me -- for my own understnading what's the point of `coverDateConfirmed`? Is it now just effectively a nice way to check `requestedCoverStartDate` is not undefined?